### ISSUE TITLE
Azure Function Event Hub trigger stops working (App Service plan)

### ISSUE THREAD

**Issue by xelossan** [Azure/Azure-Functions#182]
I've got a problem with Azure Function with Event Hub trigger, running on App Service Plan (Standard S1). Everything worked fine for a couple of days, and then it stopped processing any event hub messages.

After opening Function editor in portal and manually triggering the Function using Run button, it processed events and started working again (I've checked that **everything was fine on February 16th, 15:32 UTC**).
Next time I checked **(February 17th, 08:52 UTC), it didn't work**. There were events incoming to the event hub, but function wasn't triggered. I've run it manually one more time, **and at 13:30 UTC it didn't work again**. I haven't run it since then, and I can confirm that right now it is not working.

The app name is **cx-in-functions**, function **cxin-hub-trigger**, bindings:

```
{
  "bindings": [
    {
      "type": "eventHubTrigger",
      "name": "message",
      "direction": "in",
      "path": "cxin-hub",
      "connection": "cxin-hub",
      "consumerGroup": "$Default"
    },
    {
      "type": "notificationHub",
      "name": "notification",
      "hubName": "cxin-notification-hub",
      "connection": "cxin-notification-hub_NOTIFICATIONHUB",
      "direction": "out"
    },
    {
      "type": "table",
      "name": "respondentsTable",
      "tableName": "respondents",
      "take": 50,
      "connection": "cxask_STORAGE",
      "direction": "in"
    }
  ],
  "disabled": false
}
```

Regards,
Tomasz

**Comment by tohling** [comment]
Apologies for the delayed response.  I just checked and as of 03/29/2017 4:10 PM PS, it seems that you Function App named **cx-in-functions** currently only has 1 HTTP-Trigger type.  This change was made on 03/19/2017 10:44:28 AM UTC.

Kindly let us know when you have created a new Function or Function App with the repro and we will take a look.

**Comment by ghclo** [comment]
@xelossan I met the same issue. I have Event Hub triggered function app running on App Service Plan (Standard S2), always_on is enabled.  Function app was running good, but at some point, it just stops processing any event hub messages. There were events incoming to the event hub, but function wasn't triggered. 
It will resume working once I restart the function app, but after a while, it stopped again. 

I wonder is your issue solved and may I know what's the cause and how did you get it solve?

**Comment by dazfuller** [comment]
Just had the same issue as well.  Been working fine for a few weeks (normally triggers only once a day) and then just stopped today. Restarted the function app and everything is working again.  I've trawled through App Insights and there's no reported errors or issues.

**Comment by tohling** [comment]
@dazfuller, @xelossan  thank you for reaching out. This issue has been closed. Kindly open a new issue at https://github.com/Azure/azure-functions-host/issues/new and provide some more details on your active Function App/Function so that we can investigate further.
